### PR TITLE
Add toggle variable action

### PR DESCRIPTION
**Pull request by minad**

### CODE REVIEW

File: embark.el [hunk 1]
```diff
@@ -3557,6 +3558,14 @@ Returns the new name actually used."
   (interactive "SVariable: ")
   (insert (string-trim (pp-to-string (symbol-value var)))))
 
+(defun embark-toggle-variable-value (var)
+  "Insert value of VAR."
```
Comment (oantolin) [inline embark.el:3562]: Surely a less misleading docstring is possible. 😛

**Comment by oantolin** [comment]
Oh boy, I have a bunch of work to do both here and on orderless! Thanks for making the pull requests. I'll slowly get to them.

**Comment by oantolin** [comment]
With the docstring change I'd say this is ready to merge. We can deal with fixing the restart behavior separately; plus, it affects more than this action.

File: embark.el [hunk 2]
```diff
@@ -3557,6 +3558,14 @@ Returns the new name actually used."
   (interactive "SVariable: ")
   (insert (string-trim (pp-to-string (symbol-value var)))))
 
+(defun embark-toggle-variable-value (var)
+  "Toggle value of boolean variable VAR."
+  (interactive "SVariable: ")
+  (let ((val (symbol-value var)))
+    (unless (memq val '(nil t))
+      (error "Not a boolean variable"))
```
Comment (oantolin) [inline embark.el:3566]: I think this should be a user-error, no?

**Comment by minad** [comment]
> With the docstring change I'd say this is ready to merge. We can deal with fixing the restart behavior separately; plus, it affects more than this action.

Okay. Ready from my side. I've removed the restart for now, but I will open an issue to keep this in mind.

**Comment by oantolin** [comment]
Great, thanks! I'll actually use this quite a bit, since I often toggle `sentence-end-double-space`.

**Comment by minad** [comment]
Yes, it is a nice feature. Btw, I've stolen this from Drew's :ice_cube:. See https://lists.gnu.org/archive/html/emacs-devel/2022-09/msg00790.html. In ten years or so we will be feature complete. :smile:

**Pull request merged**